lib/extractor/default.nix

```diff
@@ -106,15 +116,19 @@ let
       # only use selected interpreters
       in optionalString (pyVersions == [] || elem v pyVersions) ''
         echo "extracting metadata for python${v}"
-        out_file=$out/python${v}.json ${py}/bin/python -c "${setuptools_shim}" install &> $out/python${v}.log || true
+        SETUPTOOLS_USE_DISTUTILS=stdlib out_file=$out/python${v}.json ${py}/bin/python -c "${setuptools_shim}" install &> $out/python${v}.log || true
       ''
     ))}
   '';
+  # SETUPTOOLS_USE_DISTUTILS=stdlib: Restore old setuptools behaviour (since
```

Comment:
Sorry for nitpicking, but wouldn't it be better to have the comment before the first variable occurrence instead of between them? (Ignore me if you already considered this.)

Reply:
I don't care either way, as long as it's not inside the 'script =' nixlang block.
Before both occurrences it is.